modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -126,6 +126,20 @@
 			<param index="0" name="value" type="Variant" />
 			<param index="1" name="type" type="Variant" />
 			<description>
+				Returns [code]true[/code] if [param value] is an instance of [param type]. The [param type] value must be one of the following:
+				- A constant from the [enum Variant.Type] enumeration, for example [constant TYPE_INT].
+				- An [Object]-derived class which exists in [ClassDB], for example [Node].
+				- A [Script] (you can use any class, including inner one).
+				Unlike the right operand of the [code]is[/code] operator, [param type] can be a non-constant value. The [code]is[/code] operator supports more features (such as typed arrays) and is more performant. Use the operator instead of this method if you do not need dynamic type checking.
```

Comment:
I shouldn't have written about performance, especially without tests. This is too implementation dependent. But I'm surprised that `is_instance_of()` is faster, if my tests are correct.